sdk/iOS/OoyalaSkinSDK/OOConstant.m

```diff
@@ -8,5 +8,5 @@
 
 #import "OOConstant.h"
 
-NSString *const OO_SKIN_VERSION = @"v4.45.0";
+NSString *const OO_SKIN_VERSION = @"v4.46.0";
```

Comment:
This should be `4.47` since these changes will be in the next release

Reply:
Done